src/css/job-posts.css

```diff
@@ -0,0 +1,299 @@
+.job-posts {
+  background:var(--COLOR_DARK_BLUE) center;
+}
+
+.job-posts__container {
+  padding-top: 95px;
+  padding-bottom: 70px;
+}
+
+.job-posts__title {
+  margin-bottom: 50px;
+  font-weight: var(--FONT_WEIGHT_700);
+  color: var(--COLOR_WHITE);
+  font-size: var(--FONT_SIZE_48);
+  letter-spacing: 0.05em;
+  line-height: 75px;
+}
+
+.job-posts__card-wrapper {
+  display: flex;
+  margin-bottom: 60px;
+  gap: 30px;
+  justify-content: center;
+}
+
+.card {
+  width: 370px;
+  height: 250px;
+  border-radius: 20px;
+  padding: 15px 30px 20px 20px;
+  background: var(--COLOR_WHITE);
+}
+
+.card:first-child {
+  background: rgba(238, 79, 60, 0.5);
+}
+
+.card-content__top {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.card-content__left-side {
+  display: flex;
+  align-items: center;
+}
+
+.card-content__card-circle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 55px;
+  width: 55px;
+  border-radius: 50px;
+  background-color: var(--COLOR_WHITE);
+  margin-right: 20px;
+  box-shadow: 0px 4px 50px rgba(153, 153, 153, 0.25);
+}
+
+.card-content__title {
+  font-size: var(--FONT_SIZE_20);
+}
+
+.card-content__right-side {
+  position: relative;
+  margin: 10px -5px 0 0;
+  display: flex;
+  z-index: 4;
+}
+
+.right-side__first-circle {
+  position: relative;
+  z-index: 1;
+  left: 39px;
+  max-height: 44px;
+}
+
+.right-side__second-circle {
+  position: relative;
+  z-index: 2;
+  left: 19px;
+  max-height: 44px;
+}
+
+.right-side__third-circle {
+  position: relative;
+  z-index: 3;
+  text-align: center;
+  max-height: 42px;
+}
+
+.avatar-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.card-content__middle {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  line-height: 25px;
+}
+
+.card-content__middle_title {
+  margin-bottom: 12px;
+  font-weight: var(--FONT_WEIGHT_700);
+  font-size: var(--FONT_SIZE_24);
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.card-content__middle_subtitle {
+  margin-bottom: 36px;
+}
+
+.card-content__bottom {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card-content__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 35px;
```

Comment:
I think it would be simpler not to write height here, but to use paddings. This will also help you to fix vertical alignment in buttons and to keep space around text inside on smaller screens.

Reply:
changed to padding: 8px 16px;